app/controllers/data/data.controller.js

```diff
@@ -0,0 +1,23 @@
+'use strict'
+
+/**
+ * Controller for /data endpoints
+ * @module DataController
+ */
+
+const Boom = require('@hapi/boom')
```

Comment:
Tiny thing! We've got [a convention](https://github.com/DEFRA/water-abstraction-team/blob/main/coding_conventions.md#top-of-js-files) that splits external modules from internal ones with a new line.

```suggestion
const Boom = require('@hapi/boom')

```